Frontend/src/components/mentor/mentor-profile/MentorProfile.jsx

```diff
@@ -206,6 +268,16 @@ function MentorProfile() {
               </Typography>
             </div>
           </div>
+          {user.userRole != "mentor" && (
```

Comment:
https://stackoverflow.com/questions/8616483/javascript-comparison-operators-vs

probably better to use !== here instead of != 
I don't think it changes anything here but this is good for consistency